### PR TITLE
add note about running private packages from builder

### PR DESCRIPTION
**Pull request by brewn**
Signed-off-by: brewn <nbrewer@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by brewn** [comment]
https://github.com/habitat-sh/habitat/issues/4733

File: www/source/partials/docs/_dev-pkgs-share.html.md.erb [hunk 1]
```diff
@@ -83,7 +85,7 @@ $ hab start core/postgresql --channel unstable
 
 An exported tarball package contains the Habitat client/binary as well as dependencies specified by your artifact.
 
-After deploying the tarball to your target server, extract the contents to the _root directory_:
+After deploying the tarball to your target server, extract the contents to the `root` directory:
```
Comment (cnunciato) [inline www/source/partials/docs/_dev-pkgs-share.html.md.erb]: Well, hm. It isn't literally `root`, it's `/` (I actually stumbled on this verbiage when I read it for the first time, too.) Maybe we should just say "to the filesystem root directory (`/`)" so the user can match that up with the `/` in the example?

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/354448734).

**Comment by cnunciato** [comment]
Oops! Clicked the wrong button. 😄

**Pull request merged**